frontend/src/components/App.tsx

```diff
@@ -8,9 +8,12 @@ import Drawer from "./drawer/Drawer";
 import { Explore } from "./pages/explore/Explore";
 import { hot } from "react-hot-loader";
 import { Login } from "./pages/login/Login"; 
-import { Profile } from "./pages/profile/Profile";
+import Profile from "./pages/profile/Profile"
```

Comment:
```suggestion
import Profile from "./pages/profile/Profile";
```
Please add this semicolon.